### DISCUSSION TITLE
using Ormlite on Raspberry pi

### DISCUSSION THREAD

**Discussion by doronweiss** [ServiceStack/Discuss#25] [Q&A] [answered]
i build an .NET 6 application running on Raspberry Pi/Raspebian (ubuntu/debian linux) 
I use an SQLite DB (using Microsoft.Data.Sqlite), accessing the DB with regular SQL queries
I wanted to use OrmLite which i like very much for this application but the application keeps crashing saying it cannot load "sqlite.interop.dll"
I googled heavily and found some very non-trivial work-arounds requiring compilation of this DLL etc'
Did any one use the OrmLite on Raspberry pi/linux ? preferably ARM32 (ARM64 is OK too)?
please help a desperate developer :-)

**Comment by mythz** [comment, marked as answer]
If you're able to use **Microsoft.Data.Sqlite**, then using [ServiceStack.OrmLite.Sqlite.Data](https://docs.servicestack.net/ormlite/installation#sqlite) stands the best chance of working:

```xml
<PackageReference Include="ServiceStack.OrmLite.Sqlite.Data" Version="6.*" />
```

**Reply by doronweiss** [reply]
worked great, thanks!
